specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -12,7 +12,7 @@ required:
 properties:
   id:
     type: string
-    example: 'appointment'
+    example: 'appointment or 1'
```

Comment:
I think the old example was ok here, the ID has been defined as a string so 1 would be allowed but you wouldn't see an appointment with the ID "appointment or 1"